src/docker.rs

```diff
@@ -208,6 +215,40 @@ WORKDIR /home/solana
         Ok(())
     }
 
+    fn insert_client_accounts_if_present(
+        &self,
+        solana_root_path: &Path,
+        validator_type: &ValidatorType,
+    ) -> Result<String, Box<dyn Error>> {
+        match validator_type {
+            ValidatorType::Client(index) => {
+                let bench_tps_path =
+                    solana_root_path.join(format!("config-k8s/bench-tps-{index}.yml"));
+                if bench_tps_path.exists() {
+                    Ok(format!(
+                        r#"
+COPY --chown=solana:solana ./config-k8s/bench-tps-{index}.yml /home/solana/client-accounts.yml
```

Comment:
ahh we can't really do this since `bench_tps_path` is the full path on the host. whereas `./config-k8s/bench-tps-{index}.yml` is the path in the container.